dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -65,3 +65,9 @@ stderr_logfile=syslog
 {% endfor %}
 {% endif %}
 {% endif %}
+
+[eventlistener:kill_supervisor]
```

Comment:
I think the `[eventlistener:...]` section(s) should be listed immediately after the `[supervisord]` section; before any `[program:...]` sections. This applies to all supervisor config files.